src/main/java/org/opentripplanner/routing/api/request/ItineraryFilterParameters.java

```diff
@@ -58,12 +58,22 @@
    */
   public DoubleFunction<Double> transitGeneralizedCostLimit = null;
 
+  /**
+   * This value works the same way as {@link #transitGeneralizedCostLimit}, only for non-transit
+   * itineraries.
+    */
+  public DoubleFunction<Double> nonTransitGeneralizedCostLimit = null;
+
 
   private ItineraryFilterParameters() {
     this.debug = false;
     this.groupSimilarityKeepOne = 0.85;
     this.groupSimilarityKeepNumOfItineraries = 0.68;
     this.minSafeTransferTimeFactor = 0.0;
+    this.transitGeneralizedCostLimit =
+        RequestFunctions.createLinearFunction(3600, 2);
```

Comment:
This changes the default value, witch I think is nice. But, the doc in various places must be updated to reflect the new default value.